src/cpr_sdk/tools_agents/tools/search.py

```diff
@@ -63,6 +63,37 @@ def search_database(
         return all_results
 
 
+def search_for_documents_by_title(
+    title: str,
```

Comment:
nit picking: I think I'd go for something like `search_terms` or `query_string` here, rather than `title`. To me, a kwarg like `title` tells me that i need to search for titles _using a title_ rather than a regular bunch of keywords

Reply:
good point - fixed!